composer.json

```diff
@@ -41,16 +41,16 @@
         "psr/container": "^2.0",
         "psr/event-dispatcher": "^1.0",
         "psr/log": "^3.0",
-        "psr/simple-cache": "^2.0|^3.0",
-        "spiral/attributes": "^2.7|^3.0",
+        "psr/simple-cache": "^3.0",
```

Comment:
I prefer using the both versions, v2 and v3. There are a lot of packages that still use v1|v2

Reply:
The last release in the 2nd branch was 4 years ago.
Maybe it needs to be updated?

Reply:
If a dependency does not limit us, I would keep the support range as wide as possible.
There are still many Cycle users who use PHP 8.0.
Some users are gradually migrate projects to Yii 3 and Cycle packages. They asked to bring back compatibility with PSR v2 where we removed it.

Reply:
If users are moving to another framework, the transition from version 2 to version 3 of the package should not be difficult for them. 🤨

Reply:
`psr/simple-cache` is not used, so we can remove this dependency.

Maybe there are other unused dependencies?